### PR TITLE
Add beginnings of a common issue tracker interface (#504).

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
@jonathanmetzman - can you help to review.

File: src/python/issue_management/monorail/__init__.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    monorail_issue = MonorailIssue()
+    monorail_issue.itm = self._itm
+    return Issue(monorail_issue)
+
+  def get_issue(self, issue_id):
+    monorail_issue = self._itm.get_issue(issue_id)
+    if not monorail_issue:
+      return None
+
+    return Issue(self._itm.get_issue(issue_id))
```
Comment (inferno-chromium) [inline src/python/issue_management/monorail/__init__.py]: s/self._itm.get_issue(issue_id)/monorail_issue

File: src/python/issue_management/issue_tracker.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    """The issue reporter."""
+    raise NotImplementedError
+
+  @reporter.setter
+  def reporter(self, new_reporter):
+    raise NotImplementedError
+
+  @property
+  def merged_into(self):
+    """The issue reporter."""
```
Comment (inferno-chromium) [inline src/python/issue_management/issue_tracker.py]: Fix description.

Comment (oliverchang) [reply src/python/issue_management/issue_tracker.py]: done

File: src/python/issue_management/monorail/__init__.py (hunk 1 above)
Comment (oliverchang) [reply src/python/issue_management/monorail/__init__.py]: done.

**Pull request merged**